src/HealthcareApis/HealthcareApis/ChangeLog.md

```diff
@@ -19,6 +19,10 @@
 -->
 ## Upcoming Release
 
+## Version 3.0.0
```

Comment:
I'm curious about the jump to 3.0.0.
- Changing the first digit indicates a breaking change.  But, I don't think this is a breaking change.
- Changing the second digit indicates a non-breaking addition of functionality (ie, a new feature)

The latter seems most appropriate here.  In that case, 1.2.0 seems like a good version number.  Thoughts here?

Reply:
@shmartel 3.0.0 is the nuget package version released for Private Link api version: 2020-03-30. Existing versions in the file seems to be the nuget package versions for the respective api version. I referred below link on before adding this version in the change log. https://www.nuget.org/packages/Microsoft.Azure.Management.HealthcareApis/